collect/central_collector.go

```diff
@@ -199,6 +212,10 @@ func (c *CentralCollector) shutdown(ctx context.Context) error {
 				return err
 			}
 		}
+		// we have to make sure the health check says we're alive but not accepting data during shutdown
+		c.Health.Ready(receiverHealth, false)
+		c.Health.Ready(processorHealth, true)
+		c.Health.Ready(deciderHealth, true)
```

Comment:
Do we want to keep the decider running during shutdown? I think we only want the processor running since that's the one sending traces in the SpanCache

Reply:
I see. That makes sense. And yes, currently as soon as we receive the shutdown signal, the decider stops asking for more work

Reply:
Thanks, I've updated.